mrp_production_project_estimated_cost/models/product.py

```diff
@@ -20,8 +20,8 @@
 import openerp.addons.decimal_precision as dp
 
 
-class ProductTemplate(models.Model):
-    _inherit = 'product.template'
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
 
     manual_standard_cost = fields.Float(
```

Comment:
Se me olvidaba comentar que esto ya se hace " lo que había que hacer es actualizar el precio de los quants fabricados"
En este caso las OF reales, calculan costes reales, actualizan el precio unitario del quant y actualizan por tanto el campo coste de Odoo, 
El otro es para simulaciones y trasteos manuales. Que si no quieren usar lo dejan a cero y no pasa nada.